requirements.txt

```diff
@@ -1,16 +1,14 @@
 # Requirements for automatic testing and building
 # Runtime requirements are described in the spec file
 
-pip
-setuptools
 coverage
 coveralls
-ordered-set
 polib
 pylint
-requests
-six
 Sphinx == 1.7.6
 
 # Custom Red Hat module used with pylint
 https://github.com/rhinstaller/pocketlint/zipball/master
```

Comment:
Pocketlint is also in pypi now. It can be dragged in as the rest of the dependencies.

https://pypi.org/project/pocketlint/